libraryBuilds/common/build.gradle

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2015 the authors of j2objc-common-libs-e2e-test
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        // This is the build output of the plugin itself.
+        classpath fileTree(dir: '../../j2objc-gradle/build/libs', include: ['*.jar'])
+    }
+}
+
+subprojects {
+    apply plugin: 'java'
+    apply plugin: 'com.github.j2objccontrib.j2objcgradle'
```

Comment:
How about reversing ordering, so you apply j2objcgradle first? It's kind of a no-op since j2objcgradle will load the java plugin... but this will make sure that continues to work.